index.html

```diff
@@ -213,5 +213,115 @@ <h1>Skills</h1>
         </div>
       </div>
     </section>
+
+    <section id="contact-form-section">
+      <div class="logoimage">
+        <a href="index.html"
+          ><img
+            class="logoimage"
+            src="images/image_geometry_1(1).svg"
+            alt="My portfolio's logo"
+        /></a>
+      </div>
+      <div>
+        <h1 class="contact name">Get started</h1>
+        <p class="bio form-text">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna.
+        </p>
+        <form action="https://formspree.io/f/mgedpadw" method="POST">
+          <div class="form-container">
+            <label for="username"></label>
+            <input
+              type="text"
+              id="username"
+              name="user_name"
+              placeholder="Your name"
+              maxlength="30"
+              required
+            />
+          </div>
+          <div class="form-container">
+            <label for="email"></label>
+            <input
+              type="text"
+              id="email"
+              name="e-mail"
+              placeholder="Your email"
+              required
+            />
+          </div>
+          <div class="form-container">
+            <label for="message">
+              <textarea
+                name="message"
+                id="message"
+                cols="30"
+                rows="10"
+                maxlength="500"
+                placeholder="Enter your message here..."
+              ></textarea>
+            </label>
+          </div>
+          <button type="button" class="btn profilelin" id="button">
+            <span class="toprofile">Start collaboration</span>
+          </button>
```

Comment:
- [ ] Your form currently does not seem to be working and this button might be the reason why. Your button type here is `button` which is an arbitrary type with no specific functionality. In order for this button to work as a Submit button, its type needs to be `submit`. Another way you can do it is by using an `input` with type `submit` instead of a button. Kindly use whatever's feasible to make the form work 👍
- [ ]  The text is disappearing inside your button on hover and focus as you can see below
![image](https://user-images.githubusercontent.com/90556221/165409491-db95edeb-e5b5-4543-95af-707c9d393af2.png)
 That might be because you have the text inside a `span` element inside the button. Your button has a hover and focus selector specified but the span does not which means the text inside the span stays the same color even on hover. What do you think about removing the span altogether?